### DISCUSSION TITLE
Yet another "Sonoff" RF 315Mhz/433Mhz bridge

### DISCUSSION THREAD

**Discussion by Saur0o0n** [arendst/Tasmota#14210] [Ideas]
I need 315Mhz bridge - and this one is one of few available. It's sold as Sonoff, but there are not markings of it (both on the device and packaging), also there is no mention of itead webpage. It uses eWeLink app - so I guess this is the reason why it's called "Sonoff".

Anyway it looks like another ESP8282 device, but whole RF communication is made by Nuvoton n76e003at20 chip. There are not any connection to ESP chip exposed - and RX/TX pins (25/26) are connected to Nuvoton chip. Also, I assume, there are different antenas and chips for learning and sending RF signal.
![2021-12-29 15 56 15](https://user-images.githubusercontent.com/25805887/147678154-97b5002a-4fb3-4b9c-bbd7-f46bf40eb7ac.jpg)
From the left, top corner there is external transmitting 433Mhz antenna, below WiFi antenna, and 315Mhz transmitting antenna. On the bottom and top are receiving RF antennas. In the middle there is buzzer and touch sensor. USB is only for power supply.

![2021-12-29 16 25 10](https://user-images.githubusercontent.com/25805887/147678160-d3ff7d11-9512-472d-8539-549428c854cc.jpg)
![biridge](https://user-images.githubusercontent.com/25805887/147678672-351850d3-9207-4873-a9d7-e44680219cdd.jpg)

**Comment by sfromis** [comment]
Using the "Sonoff" name is pure marketing BS by the seller.

Good luck with that device. For Tasmota integration, you'll probably need a driver to handle the serial communication protocol (which you may need to decode first) to that other MCU chip. Such designs are usually with "the other" MCU as the main processor, with the ESP8266 merely being used as a "wifi modem".
